### ISSUE TITLE
WET 4.0 - Colour contrast failure: "Search and menus"

### ISSUE THREAD

**Issue by michael-milette** [wet-boew/wet-boew#7709]
Good day,

When viewed on mobile devices, the heading "Search and menus" seen at the top of the search/menu fails WCAG 2.0 Level AA minimum colour contrast ratio requirements as seen below.
## Government of Canada Usability theme

Working Example: http://wet-boew.github.io/themes-dist/theme-gcwu-fegc/docs/ref/theme-gcwu-fegc/theme-gcwu-fegc-en.html
![image](https://cloud.githubusercontent.com/assets/3808579/19673745/aff410a0-9a50-11e6-9a48-8f6ebfdacafe.png)
## Government of Canada Intranet theme

Working Example: http://wet-boew.github.io/themes-dist/theme-gc-intranet/docs/ref/theme-gc-intranet/theme-gc-intranet-en.html
![image](https://cloud.githubusercontent.com/assets/3808579/19673774/e85ea518-9a50-11e6-8c67-74ad6c3c2f83.png)
## Base theme

Working Example: http://wet-boew.github.io/themes-dist/theme-base/docs/ref/theme-base/theme-base-en.html
![image](https://cloud.githubusercontent.com/assets/3808579/19673963/84075572-9a52-11e6-9b1f-537753ec12a4.png)
## OGPL theme

Working Example: http://wet-boew.github.io/themes-dist/theme-ogpl/docs/ref/theme-ogpl/theme-ogpl-en.html
![image](https://cloud.githubusercontent.com/assets/3808579/19673822/63666a84-9a51-11e6-8689-e3aa826f8928.png)

I believe that the font in each case should be white as it currently is with the GCWeb and WET themes which seem to be correct.

Let me know if you have any questions.

Best regards,

Michael Milette

**Comment by masterbee** [comment]
@michael-milette  - good catch and thanks for this. Interested in doing a small P.R to tweak the CSS for each of themes or just raising awareness?

**Comment by michael-milette** [comment]
Hi @masterbee - I am currently implementing the https://github.com/wet-boew/wet-boew-moodle project on a voluntary basis - wish I was being paid :-).  This is an open source Moodle theme featuring the WET-BOEW 4.0 framework. As I was testing the mobile views, I just happen to notice it.

By the way, any CodeSprint days coming up in the near future?

**Comment by michael-milette** [comment]
Hi @masterbee, 

For the fix, see pull request https://github.com/wet-boew/wet-boew/pull/7720

Best regards,

Michael

**Comment by duboisp** [comment]
related to #7731
